test/unit/ajax.js

```diff
@@ -2840,6 +2840,32 @@ if ( typeof window.ArrayBuffer === "undefined" || typeof new XMLHttpRequest().re
 		} );
 	} );
 
+	QUnit.test( "jQuery.append with crossOrigin attribute", function( assert ) {
+		var done = assert.async();
+		assert.expect( 1 );
+
+		var timeout = null;
+
+		Globals.register( "corsCallback" );
+		window.corsCallback = function( response ) {
+			assert.ok( response === true, "Cors" );
```

Comment:
This could be more descriptive:
```suggestion
			assert.ok( response === true, "Callback fired" );
```